src/extension/tests/TestExtConfigSettingsHandler.py

```diff
@@ -170,17 +170,18 @@ def test_are_config_settings_valid(self):
                         self.config_public_settings_fields.include_classifications: ["Critical","Security"],
                         self.config_public_settings_fields.include_patches: ["*", "test*", "*ern*=1.2*", "kern*=1.23.45"],
                         self.config_public_settings_fields.exclude_patches: ["*", "test", "*test"],
-                        self.config_public_settings_fields.internal_settings: "<serialized-json>"
+                        self.config_public_settings_fields.internal_settings: "<serialized-json>",
+                        self.config_public_settings_fields.max_patch_publish_date: "2019-07-20T12:12:14Z"
```

Comment:
Same note about two test paths being required -- this is an optional field that's not guaranteed to be there. The behavior will vary significantly in code that's not written yet so please ensure tests account for that and existing tests that don't have it included remain with it not included. (New tests for the feature - both positive and negative cases - need to have it considered).

Reply:
The config settings on line 182 test for the value not specified